packages/zone/src/durable.js

```diff
@@ -103,7 +96,7 @@ export const makeDurableZone = (baggage, baseLabel = 'durableZone') => {
     return makeDurableZone(subBaggage, `${baseLabel}.${label}`);
```

Comment:
I like what durableStore is doing for `subZone` --- making the backing store be the stores child of the parent, and leaving the wrapping zone unstored. Amusingly, were `subZone` intended not to be collision protected, this would be a problem in that every time we call `subZone` with the same parent and label, we'd get a new subZone object identity. But since we can only do so once per incarnation anyway, this case cannot arise.

So no change suggested here. I'm commenting on it because part of the point of this PR is to more closely follow the precedent set by durable zones.